animalfacts.py

```diff
@@ -1433,6 +1434,22 @@ def animalfactsbot(reddit):
     'Hamsters range in size from the largest breed, the European hamster at 13.4 inches (34 cm) long, to the smallest, the dwarf hamster at 2 - 4 inches (5.5 - 10.5 cm) long.',
     )
 
+HAWK_FACTS = (
+    'Hawks can vary in size depending on the species.',
+    'Hawks have excellent eyesight. They can see 8 times better than humans and can locate prey from 100 feet away.',
+    'Hawks are diurnal animals (active during the day.',
```

Comment:
Missing ) here if you could please

Reply:
See 769bc04265ce52192628dfa19c5f39765a1c12ce